test/call.c

```diff
@@ -210,6 +211,13 @@ static void event_handler(struct ua *ua, enum ua_event ev,
 			}
 			break;
 
+		case BEHAVIOUR_REDIRECT:
+			ua_redirect(ua, call, 302, NULL,
+				    "sip:alice@example.com");
```

Comment:
the testcode should only use internal dummy AORs.

public DNS names should not be used, as the testcode should be able to run
in an isolated environment with no internet access...

in your case you can create 3 UAs, A, B and C.

When doing the redirect, you can redirect to C and then verify that the call was established.

Reply:
I thought that `ua_redirect()` testcase only needs to test if 3xx response is properly _send_ and that the call is then closed.  The test on what happens when baresip receives 3xx belongs to somewhere else and is an existing feature.